string map_path
---
bool success
string message
